tinymce-angular-component/src/main/ts/utils/Utils.ts

```diff
@@ -8,15 +8,27 @@
 
 import { EventEmitter } from '@angular/core';
 import { EditorComponent } from '../editor/editor.component';
-import { validEvents } from '../editor/Events';
+import { validEvents, Events } from '../editor/Events';
 
 const bindHandlers = (ctx: EditorComponent, editor: any): void => {
-  validEvents.forEach((eventName) => {
+  const ignoredEvents = parseStringProperty(ctx.ignoreEvents, []);
+  const allowedEvents = parseStringProperty(ctx.allowedEvents, validEvents).filter((event) => !ignoredEvents.includes(event)) as (keyof Events)[];
```

Comment:
Is it intended that the user can now add new events that are not in `validEvents`?

Reply:
Filter by valid events now